diff --git a/Makefile.dist b/Makefile.dist
index 61194a07..a1597d9f 100644
--- a/Makefile.dist
+++ b/Makefile.dist
@@ -6,7 +6,7 @@ CC=gcc
 CXX=g++
 
 # OpenMP support
-#CFLAGS+=-fopenmp
+CFLAGS+=-fopenmp
 
 # RawSpeed Support
 #CFLAGS+=-pthread -DUSE_RAWSPEED -I../RawSpeed -I/usr/local/include/libxml2
@@ -26,10 +26,10 @@ CFLAGS+=-DUSE_ZLIB
 LDADD+=-lz
 
 # JPEG support for lossy DNG
-#CFLAGS+=-DUSE_JPEG -I/usr/local/include
-#LDADD+=-L/usr/local/lib -ljpeg
+CFLAGS+=-DUSE_JPEG -I/usr/local/include
+LDADD+=-L/usr/local/lib -ljpeg
 # LIBJPEG8:
-#CFLAGS+=-DUSE_JPEG8
+CFLAGS+=-DUSE_JPEG8
 
 # LCMS support
 #CFLAGS+=-DUSE_LCMS -I/usr/local/include
diff --git a/Makefile.mingw b/Makefile.mingw
index f2c291e2..a266dfee 100644
--- a/Makefile.mingw
+++ b/Makefile.mingw
@@ -5,7 +5,7 @@ CC=gcc
 CXX=g++
 
 # OpenMP support
-#CFLAGS+=-fopenmp
+CFLAGS+=-fopenmp
 
 # RawSpeed Support
 #CFLAGS+=-pthread -DUSE_RAWSPEED -I../RawSpeed -I/usr/local/include/libxml2
@@ -21,8 +21,8 @@ CXX=g++
 #LDADD+=-L/usr/local/lib -ljasper
 
 # JPEG support for lossy DNG
-#CFLAGS+=-DUSE_JPEG -I/usr/local/include
-#LDADD+=-L/usr/local/lib -ljpeg
+CFLAGS+=-DUSE_JPEG -Ic:/msys64/mingw64/include
+LDADD+=-L/usr/local/lib -ljpeg
 # LIBJPEG8:
 #CFLAGS+=-DUSE_JPEG8
 
