*** LibRaw/Makefile.dist	Wed Jun 28 14:44:51 2017
--- LibRaw-new/Makefile.dist	Wed Jun 28 15:13:02 2017
***************
*** 7,11 ****
  
  # OpenMP support
! #CFLAGS+=-fopenmp
  
  # RawSpeed Support
--- 7,11 ----
  
  # OpenMP support
! CFLAGS+=-fopenmp
  
  # RawSpeed Support
***************
*** 23,28 ****
  
  # JPEG support for lossy DNG
! #CFLAGS+=-DUSE_JPEG -I/usr/local/include
! #LDADD+=-L/usr/local/lib -ljpeg
  # LIBJPEG8:
  #CFLAGS+=-DUSE_JPEG8
--- 23,28 ----
  
  # JPEG support for lossy DNG
! CFLAGS+=-DUSE_JPEG
! LDADD+=-ljpeg
  # LIBJPEG8:
  #CFLAGS+=-DUSE_JPEG8
***************
*** 37,46 ****
  
  # Demosaic Pack GPL2:
! #DPCFLAGS+=-I../LibRaw-demosaic-pack-GPL2
! #CFLAGS+=-DLIBRAW_DEMOSAIC_PACK_GPL2
  
  # Demosaic Pack GPL3:
! #DPCFLAGS+=-I../LibRaw-demosaic-pack-GPL3
! #CFLAGS+=-DLIBRAW_DEMOSAIC_PACK_GPL3
  
  
--- 37,46 ----
  
  # Demosaic Pack GPL2:
! DPCFLAGS+=-I../LibRaw-demosaic-pack-GPL2
! CFLAGS+=-DLIBRAW_DEMOSAIC_PACK_GPL2
  
  # Demosaic Pack GPL3:
! DPCFLAGS+=-I../LibRaw-demosaic-pack-GPL3
! CFLAGS+=-DLIBRAW_DEMOSAIC_PACK_GPL3
  
  
