From caa15c502b6b3388eba732f6785bdf8abac6e416 Mon Sep 17 00:00:00 2001
From: CarVac <c.lo.to.da.down.lo@gmail.com>
Date: Wed, 12 Sep 2018 21:31:15 -0400
Subject: [PATCH] change makefile

---
 Makefile.dist | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/Makefile.dist b/Makefile.dist
index cb2f159..36b2c97 100644
--- a/Makefile.dist
+++ b/Makefile.dist
@@ -1,12 +1,12 @@
 all: library all_samples
 
-#CFLAGS=-arch i386 -arch x86_64 -O3  -I. -w
-CFLAGS=-O3  -I. -w
+CFLAGS=-march=i386 -march=x86_64 -O3  -I. -w
+#CFLAGS=-O3  -I. -w
 CC=gcc
 CXX=g++
 
 # OpenMP support
-#CFLAGS+=-fopenmp
+CFLAGS+=-fopenmp
 
 # RawSpeed Support
 #CFLAGS+=-pthread -DUSE_RAWSPEED -I../RawSpeed -I/usr/local/include/libxml2
@@ -22,8 +22,8 @@ CXX=g++
 #LDADD+=-L/usr/local/lib -ljasper
 
 # JPEG support for lossy DNG
-#CFLAGS+=-DUSE_JPEG -I/usr/local/include
-#LDADD+=-L/usr/local/lib -ljpeg
+CFLAGS+=-DUSE_JPEG
+LDADD+=-ljpeg
 # LIBJPEG8:
 #CFLAGS+=-DUSE_JPEG8
 
-- 
2.17.1

