*** Makefile.dist	2018-04-15 11:21:14.849133275 -0400
--- patch/Makefile.dist	2018-04-15 11:21:38.477269777 -0400
***************
*** 1,11 ****
  all: library all_samples
  
! CFLAGS=-arch i386 -arch x86_64 -O3  -I. -w
  CC=gcc
  CXX=g++
  
  # OpenMP support
! #CFLAGS+=-fopenmp
  
  # RawSpeed Support
  #CFLAGS+=-pthread -DUSE_RAWSPEED -I../RawSpeed -I/usr/local/include/libxml2
--- 1,11 ----
  all: library all_samples
  
! CFLAGS=-march=i386 -march=x86-64 -O3  -I. -w
  CC=gcc
  CXX=g++
  
  # OpenMP support
! CFLAGS+=-fopenmp
  
  # RawSpeed Support
  #CFLAGS+=-pthread -DUSE_RAWSPEED -I../RawSpeed -I/usr/local/include/libxml2
***************
*** 21,28 ****
  #LDADD+=-L/usr/local/lib -ljasper
  
  # JPEG support for lossy DNG
! #CFLAGS+=-DUSE_JPEG -I/usr/local/include
! #LDADD+=-L/usr/local/lib -ljpeg
  # LIBJPEG8:
  #CFLAGS+=-DUSE_JPEG8
  
--- 21,28 ----
  #LDADD+=-L/usr/local/lib -ljasper
  
  # JPEG support for lossy DNG
! CFLAGS+=-DUSE_JPEG
! LDADD+=-ljpeg
  # LIBJPEG8:
  #CFLAGS+=-DUSE_JPEG8
  
***************
*** 35,46 ****
  #LDADD+=-L/usr/local/lib -llcms2
  
  # Demosaic Pack GPL2:
! #DPCFLAGS+=-I../LibRaw-demosaic-pack-GPL2
! #CFLAGS+=-DLIBRAW_DEMOSAIC_PACK_GPL2
  
  # Demosaic Pack GPL3:
! #DPCFLAGS+=-I../LibRaw-demosaic-pack-GPL3
! #CFLAGS+=-DLIBRAW_DEMOSAIC_PACK_GPL3
  
  
  DCRAW_LIB_OBJECTS=object/dcraw_common.o object/libraw_cxx.o object/libraw_datastream.o object/libraw_c_api.o object/dcraw_fileio.o object/demosaic_packs.o
--- 35,46 ----
  #LDADD+=-L/usr/local/lib -llcms2
  
  # Demosaic Pack GPL2:
! DPCFLAGS+=-I../LibRaw-demosaic-pack-GPL2
! CFLAGS+=-DLIBRAW_DEMOSAIC_PACK_GPL2
  
  # Demosaic Pack GPL3:
! DPCFLAGS+=-I../LibRaw-demosaic-pack-GPL3
! CFLAGS+=-DLIBRAW_DEMOSAIC_PACK_GPL3
  
  
  DCRAW_LIB_OBJECTS=object/dcraw_common.o object/libraw_cxx.o object/libraw_datastream.o object/libraw_c_api.o object/dcraw_fileio.o object/demosaic_packs.o
